crates/libs/result/src/error.rs

```diff
@@ -1,10 +1,58 @@
 use super::*;
+use crate::HRESULT;
 use core::ffi::c_void;
 
+#[allow(dead_code)] // for cases that are not yet stabilized; see below
+mod winerror {
+    use crate::HRESULT;
+
+    // We define these here for non-Windows platforms.
+    // It's just as easy to use them on Windows, too.
```

Comment:
This seems to favor non-Windows. Why not just keep the bindings and use them "just as easily" on non-Windows platforms? Much of the value of the generating bindings is not having to maintain these manual definitions.

Reply:
Yeah, that makes sense.  The problem really arises from how easy it is to accidentally use something that is not available on other platforms.  I can reduce this to focus on the DLL imports that are not available, such as GetErrorInfo.

Reply:
Fortunately, it's easy enough to test - just make sure a linux yml workflow is exercising `windows-core` sufficiently. That would be preferable to manually defining bindings.